Microsoft.Toolkit.Uwp.UI.Controls/TextToolbar/ToolbarFormatterActiveConverter.cs

```diff
@@ -11,15 +11,15 @@ namespace Microsoft.Toolkit.Uwp.UI.Converters
     /// <summary>
     /// Compares if Formats are equal and returns bool
     /// </summary>
-    public class ToolbarFormatActiveConverter : IValueConverter
+    public class ToolbarFormatterActiveConverter : IValueConverter
```

Comment:
I think we should just generalize this now and move this to the Converters folder in the Uwp project and make it a general 'TypeToObjectConverter'.

Was going to say make it a subclass of `BoolToObjectConverter`, but I don't think that quite works as constructed, it's probably it's own class for now, unless we restructure.

Basically though the converter would provide one value if the Type matches and otherwise the other. In the sample above this would be Visible or Collapsed.

Reply:
I agree. I'll make the change.

Reply:
Just pushed it. Let me know what you think.